docs/webhooks.rst

```diff
@@ -0,0 +1,10 @@
+Registering webhooks to be notified of helpesk events
+-----------------------------------------------------
+
+You can register webhooks to allow third party apps to be notified of helpdesk events. Webhooks can be registered in one of two ways:
+
+1. Setting comma separated environement variables; ``HELPDESK_NEW_TICKET_WEBHOOK_URLS``& ``HELPDESK_FOLLOWUP_WEBHOOK_URLS``.
+
+2. Adding getter functions to your ``settings.py``. These should return a list of strings (urls); ``HELPDESK_GET_NEW_TICKET_WEBHOOK_URLS`` & ``HELPDESK_GET_FOLLOWUP_WEBHOOK_URLS``.
+
```

Comment:
Would be good to add the HELPDESK_WEBHOOK_TIMEOUT setting description here as well.